### DISCUSSION TITLE
Why ID.unique() returns "unique()" ?

### DISCUSSION THREAD

**Discussion by jostney** [appwrite/appwrite#7355] [Q&A] [answered]
### 👟 Reproduction steps

The current version is  "node-appwrite": "^11.1.0",

var unique_id = sdk.ID.unique();

### 👍 Expected behavior

I was assuming node.js sdk will return unique id 

### 👎 Actual Behavior

Unfortunately returns string  "unique()" 

### 🎲 Appwrite version

Different version (specify in environment)

### 💻 Operating system

MacOS

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment, marked as answer]
Thanks for creating this issue! 🙏 The `unique()` string is a special keyword Appwrite uses to generate an ID for you. See https://github.com/appwrite/appwrite/blob/5cd94041dc1037f53f62d6afb78528021ece3fa1/app/controllers/api/databases.php#L2812.

Also, we wanted to include this in the SDK because before we had these helper methods, people were creating constants in their projects for these values so we wanted to help save a step.
